resources/views/site/shared/_featuredRelated.blade.php

```diff
@@ -17,9 +17,26 @@
     if (($behavior ?? '') === 'relatedSidebar') {
         $listingVariation .= ' m-listing--dynamic';
     }
+
+    // Get featured related items
+    $isFeatured = false;
+    $featuredRelated = collect($item->getFeaturedRelated())->pluck('item');
+
+    $featuredRelatedIds = $featuredRelated->pluck('id');
+
+    // Get auto related items & evaluate if they are featured
+
+    $autoRelated = collect($item->related($item->id))->unique('id')->filter();
+
+    // Remove featured related items from auto related items
+    if ($featuredRelatedIds->isNotEmpty()) {
+        $autoRelated = $autoRelated->reject(function ($relatedItem) use ($featuredRelatedIds) {
+            return ($relatedItem !== null && ($featuredRelatedIds->contains($relatedItem->id) || $featuredRelatedIds->contains($relatedItem->datahub_id)));
+        });
+    }
```

Comment:
Same as previous comment, move this into the `show()` methods of the applicable controllers and pass an `$autoRelated` array to the view.